evap/evaluation/models.py

```diff
@@ -470,7 +470,10 @@ def evaluation_ends_soon(self):
 
     @property
     def days_until_evaluation(self):
-        return (self.vote_start_datetime.date() - date.today()).days
+        days_left = (self.vote_start_datetime.date() - date.today()).days
+        if days_left == 0 and self.vote_start_datetime < datetime.now():
+            return -1
+        return days_left
 
     def is_user_editor_or_delegate(self, user):
         if self.contributions.filter(can_edit=True, contributor=user).exists():
```

Comment:
by the way is there a reason why students don't have a "starts today" or "starts tomorrow" badge on their index page?

Reply:
no. is not really important but can be added.

Reply:
will do so in the distribution_bars branch where the student index will be changed anyways